### PR TITLE
Gripper controller adaptation

### PR DESCRIPTION
**Pull request by livanov93**
PR info:

- Change gripper controller to make it visible to moveit
- Other fixes that handle the driver stability

### CODE REVIEW

File: kortex2_bringup/config/kortex_controllers.yaml [hunk 1]
```diff
@@ -12,8 +12,10 @@ controller_manager:
       type: kortex2_controllers/TwistController
 
     # The gripper controller just relays position commands
```
Comment (destogl) [inline kortex2_bringup/config/kortex_controllers.yaml:14]: should we remove this commented lines?

File: kortex2_bringup/config/kortex_controllers.yaml [hunk 2]
```diff
@@ -52,8 +54,13 @@ streaming_controller:
       - twist.angular.y
       - twist.angular.z
 
-hand_controller:
+#hand_controller:
+#  ros__parameters:
+#    joints:
+#      - finger_joint
+#    interface_name: position
+
```
Comment (destogl) [inline kortex2_bringup/config/kortex_controllers.yaml:62]:
```suggestion
```

File: kortex2_driver/src/hardware_interface.cpp [hunk 3]
```diff
@@ -714,8 +722,9 @@ return_type KortexMultiInterfaceHardware::read()
   if (first_pass_)
   {
     first_pass_ = false;
+    feedback_ = base_cyclic_.RefreshFeedback();
   }
-  feedback_ = base_cyclic_.RefreshFeedback();
+  //  feedback_ = base_cyclic_.RefreshFeedback();
```
Comment (destogl) [inline kortex2_driver/src/hardware_interface.cpp:727]: Is this obsolete now? If so I would delete it...

**Review by destogl — APPROVED** [review]
Just few small comments. You decide how to proceed before merge.

File: kortex2_driver/src/hardware_interface.cpp (hunk 3 above)
Comment (livanov93) [reply kortex2_driver/src/hardware_interface.cpp:727]: Reminder for now!

**Pull request merged**